pycbc/fft/fftw_pruned.py

```diff
@@ -12,9 +12,21 @@
 twiddle factors.
 """
 from __future__ import absolute_import
-import numpy, weave, ctypes, pycbc.types
+import numpy, ctypes, pycbc.types
 from pycbc import WEAVE_FLAGS
 from pycbc.libutils import get_ctypes_library
+import logging
+from .fftw_pruned_cython import second_phase_cython, fast_second_phase_cython
```

Comment:
similar question here, why do these need to be in a separate module? Why not just leave the cython functions here?